src/ensembl/src/shared/components/breadcrumbs/Breadcrumbs.scss

```diff
@@ -0,0 +1,23 @@
+@import 'src/styles/common';
+
+.breadcrumbs {
+  color: $black;
+  font-weight: $light;
+
+  .breadcrumb{
+    margin-right: 15px;
+  }
+
+  .breadcrumb::after { 
```

Comment:
`.breadcrumb:not(:last-child)::after`? Then there will be no reason to reset the content on line 20.

Reply:
That's done